app/main.py

```diff
@@ -1 +1,35 @@
-#  write your code here
+class Animal:
+    def __init__(self, name, appetite, is_hungry=True):
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self):
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self):
+        if self.is_hungry:
+            print(f"Eating {self.appetite} food points...")
+            self.is_hungry = False
+            return self.appetite
+        return 0
+
+class Cat(Animal):
+    def __init__(self, name, is_hungry=True):
+        super().__init__(name, 3, is_hungry)
+
+    def catch_mouse(self):
+        print("The hunt began!")
```

Comment:
The `catch_mouse` method in the `Cat` class does not change the state of the object or interact with other methods. Consider adding functionality that affects the `Cat`'s state or interacts with its attributes, if required by the task description.